dbusmock/templates/bluez5.py

```diff
@@ -76,7 +90,7 @@ def AddAdapter(self, device_name, system_name):
             # Audio/Video Remote Control Profile (target)
             '0000110c-0000-1000-8000-00805f9b34fb',
         ], variant_level=1),
-        'Discoverable': dbus.Boolean(True, variant_level=1),
+        'Discoverable': dbus.Boolean(False, variant_level=1),
```

Comment:
This is a good idea, but I would also change the `Discovering` variable just below, and do that in a separate commit, with an explanation that this is the way BlueZ operates nowadays.